documentation/movies.workspace.adoc

```diff
@@ -702,6 +702,20 @@ MATCH path = (person)-[:ACTED_IN]->(m)<-[:DIRECTED]-(d)
 RETURN path;
 ----
 
+To aid query performance, you can add indexes to your data.
+(The dataset you create in this guide is very small, but it is good practice to use indexes as they can greatly improve query performance when the datasets are larger.)
+
+You already have two indexes because of the constaints created in the beginning of the `MERGE` statement, see link:https://neo4j.com/docs/cypher-manual/current/constraints/[Cypher Manual -> Constraints^] for more information.
```

Comment:
How about moving the information about already existing indexes below when you create them? That way users will be able to see the process first.

Maybe in a note 

e.g. 

[NOTE]
The database already contained 2 indexes on `Movie.title` and `Person.name`, they were automatically created with the constraints at the beginning of the `MERGE` statement.